pkg/controller/deployment/cert_manager_controller_deployment.go

```diff
@@ -67,9 +67,6 @@ func NewCertManagerControllerStaticResourcesController(operatorClient v1helpers.
 	).AddKubeInformers(kubeInformersForTargetNamespace)
 }
 
-// +kubebuilder:rbac:groups=operator.openshift.io.openshift.io,resources=certmanagers,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Is this updated in any of cr containing rbac configurations? Or is it to avoid clusterrole for the operator.openshift.io.openshift.io group?

Reply:
Already addressed in `certmanager_controller.go`.